test/integration/test_groundlight.py

```diff
@@ -336,3 +336,44 @@ def test_submit_numpy_image(gl: Groundlight, detector: Detector):
     assert str(_image_query)
     assert isinstance(_image_query, ImageQuery)
     assert is_valid_display_result(_image_query.result)
+
+
+@pytest.mark.skipif(MISSING_NUMPY or MISSING_PIL, reason="Needs numpy and pillow")  # type: ignore
+def test_detector_improvement(gl: Groundlight, detector: Detector):
+    # test that we get confidence improvement after sending images in
+    # Pass two of each type of image in
+    import time
+
+    from PIL import Image
+
+    def submit_noisy_image(image):
```

Comment:
So we want some way to randomly generate small image variations - numpy can add noise, and that's a good way.  But without numpy what can we do?  PIL/pillow has a number of methods which we can use as augmentations. How about some small random crop?  Or a small tweak to brightness?